staging/src/k8s.io/apimachinery/pkg/util/version/version.go

```diff
@@ -85,6 +86,46 @@ func parse(str string, semver bool) (*Version, error) {
 	return v, nil
 }
 
+// HighestSupportedVersion returns the highest supported version
+func HighestSupportedVersion(versions []string) (*Version, error) {
```

Comment:
Add the following comments:
This function assumes that the highest supported version must be v1.x.

Reply:
ack

Reply:
done in 42356bfbb3e